src/test/unit/platform.h

```diff
@@ -42,6 +42,9 @@
 #define USE_SERVOS
 #define USE_TRANSPONDER
 
+#define USE_ITERM_RELAX
```

Comment:
In order to keep unit tests as much isolated as possible, I think these should be defined in `<test name>_DEFINES` in `src/test/Makefile` instead of in here.

Reply:
Done